packages/vercel-ai-provider/src/edgedb-chat-settings.ts

```diff
@@ -70,6 +70,8 @@ export interface QueryContext {
 export interface EdgeDBChatConfig {
   provider: string;
   fetch: FetchFunction;
+  baseURL: string;
```

Comment:
If we're going to have this be either an explicit URL or something falsy, I'd rather it be:

```suggestion
  baseURL: string | null;
```

Instead of relying on `Boolean("") === false` behavior.